packages/web3-eth/src/index.js

```diff
@@ -138,8 +140,18 @@ var Eth = function Eth() {
     this.personal = new Personal(this.currentProvider);
     this.personal.defaultAccount = this.defaultAccount;
 
+    var ContractWrapper = function ContractWrapper() {
```

Comment:
The problem is that we want the "class" to be "Contract" not "ContractWrapper" when somebody logs the `web3.eth.contract` object.

I also dopant fully understand this wrapper.

Reply:
The problem the wrapper solves:

Given two instances of `eth`, called `eth1` and `eth2`, and two contract instances, `contract1`, created by calling `new eth1.Contract(...)` and `contract2` created by calling `new eth2.Contract(...)`, we want to ensure that calling `eth1.setProvider` does not mutate `contract2.currentProvider`.

Reply:
Also, it's trivial to rename the `ContractWrapper` type to `Contract`. I can do that if need be. I just wanted to avoid the namespace collision w/ the `Contract` type defined in the `web3-eth-contract` module.

Reply:
Yes please change to `Contract`